### PR TITLE
Feature / TEC-3755 disable v1 styles

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** This disabled the enqueueing of v1 styles to provide a clean slate for styling the whole Event Single view

**Ticket:** https://theeventscalendar.atlassian.net/browse/TEC-3755

**Screenshots:** see https://github.com/the-events-calendar/the-events-calendar/pull/3429 and others

### CODE REVIEW

File: src/Tribe/Views/V2/Assets.php [hunk 1]
```diff
@@ -451,9 +454,9 @@ public function register() {
 	 */
 	public function disable_v1() {
 		// Dont disable V1 on Single Event page
-		if ( tribe( Template_Bootstrap::class )->is_single_event() ) {
-			return;
-		}
+		// if ( tribe( Template_Bootstrap::class )->is_single_event() ) {
+		// 	return;
+		// }
```
Comment (paulmskim) [inline src/Tribe/Views/V2/Assets.php:459]: let's remove this?

File: src/resources/postcss/views/skeleton/_single-event.pcss [hunk 2]
```diff
@@ -4,9 +4,28 @@
...
  */
-	
+ 
+/* Hide a bunch of elements (from v1 styles) */
+.single-featured-image-header,
+#tribe-events-header,
+.tribe-events-divider,
+.recurring-info-tooltip,
+.tribe-events-ajax-loading
+	{
```
Comment (paulmskim) [inline src/resources/postcss/views/skeleton/_single-event.pcss:14]:
```suggestion
.single-featured-image-header,
#tribe-events-header,
.tribe-events-divider,
.recurring-info-tooltip,
.tribe-events-ajax-loading {
```

File: src/resources/postcss/views/skeleton/_single-event.pcss [hunk 3]
```diff
@@ -4,9 +4,28 @@
...
+
 #tribe-events-pg-template {
+	margin: 0 auto;
 	max-width: 1048px;
+	padding: var(--spacer-8) var(--spacer-4) var(--spacer-4);
+}
+
+.tribe-events-before-html,
+.tribe-events-after-html {
+	p {
```
Comment (paulmskim) [inline src/resources/postcss/views/skeleton/_single-event.pcss:26]:
```suggestion
.tribe-events-before-html,
.tribe-events-after-html {

	p {
```

File: src/resources/postcss/views/skeleton/_single-event.pcss [hunk 4]
```diff
@@ -4,9 +4,28 @@
  * @package TribeEventsCalendar
  *
  */
-	
+ 
+/* Hide a bunch of elements (from v1 styles) */
```
Comment (paulmskim) [inline src/resources/postcss/views/skeleton/_single-event.pcss:8]: are we still hiding v1 styles in v2? or are we removing v1 styles?

**Review by paulmskim — CHANGES_REQUESTED** [review]
a few small things and one question

File: src/resources/postcss/views/skeleton/_single-event.pcss (hunk 4 above)
Comment (alliuca) [reply src/resources/postcss/views/skeleton/_single-event.pcss:8]: @paulmskim those are some elements that were hidden in v1 and they're still not appearing in the v2 designs, even though they're part of the HTML output

Comment (paulmskim) [reply src/resources/postcss/views/skeleton/_single-event.pcss:8]: Gotcha, sounds good.

**Review by paulmskim — APPROVED** [review]

**Pull request merged**